x-pack/plugins/observability_solution/slo/public/pages/slo_details/components/groupings/slo_grouping_value_selector.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonIcon, EuiComboBox, EuiComboBoxOptionOption, EuiCopy } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { SLOWithSummaryResponse } from '@kbn/slo-schema';
+import React, { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import useDebounce from 'react-use/lib/useDebounce';
+import { SLOS_BASE_PATH } from '../../../../../common/locators/paths';
+import { useFetchSloInstances } from '../../hooks/use_fetch_slo_instances';
+import { useGetQueryParams } from '../../hooks/use_get_query_params';
+
+interface Props {
+  slo: SLOWithSummaryResponse;
+  groupingKey: string;
+  value?: string;
+}
+
+interface Field {
+  label: string;
+  value: string;
+}
+
+export function SLOGroupingValueSelector({ slo, groupingKey, value }: Props) {
+  const isAvailable = window.location.pathname.includes(SLOS_BASE_PATH);
```

Comment:
💬 This page can be displayed from the dashboard flyout, it's a little bit tricky to handle correctly so I've decided to disable the selection in this case.